src/js/common/schemaform/SaveInProgressIntro.jsx

```diff
@@ -18,13 +19,14 @@ export default class SaveInProgressIntro extends React.Component {
         const savedAt = this.props.lastSavedDate
           ? moment(this.props.lastSavedDate)
           : moment.unix(savedForm.last_updated);
+        const expirationDate = moment(savedAt).add(30, 'days');
```

Comment:
I added an issue to change the backend so we can update it here later: https://github.com/department-of-veterans-affairs/vets.gov-team/issues/4050